code/modules/projectiles/projectile.dm

```diff
@@ -15,30 +16,54 @@
 	generic_canpass = FALSE
 	blocks_emissive = EMISSIVE_BLOCK_GENERIC
 	layer = MOB_LAYER
-	//The sound this plays on impact.
+	/// The sound this plays on impact.
 	var/hitsound = 'sound/items/weapons/pierce.ogg'
-	var/hitsound_wall = ""
+	/// Sound played when the projectile hits a wall
+	var/hitsound_wall
 
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
-	var/def_zone = "" //Aiming at
-	var/atom/movable/firer = null//Who shot it
-	var/datum/fired_from = null // the thing that the projectile was fired from (gun, turret, spell)
-	var/suppressed = FALSE //Attack message
-	var/yo = null
+	/// Zone at which the projectile was aimed at
+	var/def_zone = ""
+	/// Atom which shot the projectile
+	var/atom/movable/firer = null
+	/// The thing that the projectile was fired from (gun, turret, spell)
+	var/datum/fired_from = null
+	/// One of three suppression states: NONE displays the hit message and produces a loud sound,
+	/// QUIET makes a quiet sound and only lets the victim know they've been shot, and VERY only makes a very quiet sound with no messages
+	var/suppressed = SUPPRESSED_NONE
+	/// Original clicked target
+	var/atom/original = null
+	/// Initial target x coordinate offset of the projectile
 	var/xo = null
-	var/atom/original = null // the original target clicked
-	var/turf/starting = null // the projectile's starting turf
+	/// Initial target y coordinate offset of the projectile
+	var/yo = null
```

Comment:
I would very much recommend going through some of these vars and marking them as `VAR_FINAL` (or protected/private if possible) if they're internal stuff, it would make it so much easier for future work on projectiles (being able to see at a glance which vars you can override and which you should not mess with)